code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -566,6 +568,9 @@ public PlayerCharacter(boolean load, Collection<Campaign> loadedCampaigns)
 				refContext.constructNowIfNecessary(GlobalModifiers.class,
 					"Global Modifiers");
 		globalModifierFacet.set(id, gm);
+		controller =
+				refContext.constructNowIfNecessary(CodeControl.class,
+					"Controller");
 
 		//Do BilSet first, since required by Race
```

Comment:
Bilset? Should that be bioset? Not part of the submission, but figured I'd point it out.